include/QtNodes/internal/Definitions.hpp

```diff
@@ -13,7 +13,7 @@
 
 namespace QtNodes
 {
-Q_NAMESPACE_EXPORT(NODE_EDITOR_PUBLIC)
+NODE_EDITOR_PUBLIC Q_NAMESPACE
```

Comment:
Does it actually work? I have no chance of testing it on an older versions of Qt on Windows

Reply:
Sorry, I only have a Win10 environment。Under Win10 there is no problem.

https://github.com/paceholder/nodeeditor/pull/

Here are the results of my tests on various versions of qt under Win10 and running the example